src/components/ItemList.svelte

```diff
@@ -194,8 +196,11 @@
                       clickable
                     />
                   {:else}
-                    <a href={getItemURL(appName, itemType, item.name)}
-                      >{@html highlightSearch(item.name, search)}</a
+                    <a
+                      href={getItemURL(appName, itemType, item.name)}
+                      style={itemType === "third-party data"
+                        ? "pointer-events: none"
```

Comment:
make the anchor tag non-clickable for third-party metrics because they do not have their own page like Glean metrics yet.